test-cases/tests/installation/a30-validate-installation-of-rhoam-addon-and-integration-with-ld.md

```diff
@@ -51,6 +51,14 @@ Host prefix: /26
 
 9. Click on `Create cluster` (cluster creation takes ~40 minutes)
 
+Run following command:
+
+```
+ocm get subs --parameter search="cluster_id = '<your-cluster-ID>'" | jq -r .items[0].metrics[0].health_state
```

Comment:
```suggestion
# Copy your cluster's name from OCM UI ("test-ldap-idp" by default) and assign it to the env var CLUSTER_NAME
CLUSTER_NAME=<your-cluster-name>
# Get cluster's CID
CID=$(ocm get clusters --parameter search="display_name like '$CLUSTER_NAME'" | jq -r '.items[0].id')
ocm get subs --parameter search="cluster_id = '${CID}'" | jq -r .items[0].metrics[0].health_state
```

I copied those commands from lines 77-80, so if you agree with the suggestion, those lines below can be deleted.

Reply:
Much nicer, I will do as suggested.